x-pack/plugins/apm/public/components/routing/templates/settings_template.tsx

```diff
@@ -100,13 +100,13 @@ function getTabs({
       hidden: !canAccessML,
     },
     {
-      key: 'customize-ui',
```

Comment:
imo since `Customize App` thing hasn't happened shouldn't we move the `custom_link` one level up and remove `customize_ui and make other changes accordingly ?

![Screenshot 2021-12-10 at 4 14 16 PM](https://user-images.githubusercontent.com/2223557/145565544-f74d2a78-a5dc-4dfb-b653-cb78d579229e.png)
`

Reply:
Sounds like a good idea 👍

Reply:
Should we do it in the same PR, or open a separate issue for that?

Reply:
Fine to do it in this PR

Reply:
@sqren made the change in context of removing  `customize-ui` can you take a look at it.